service-commands/scripts/utils.sh

```diff
@@ -197,7 +197,7 @@ setup_zsh() {
 	fi
 	cp $zshenv ~/.zshenv.tmp
 	IP=$(get_ip_addr $provider $name)
-	echo "export AUDIUS_REMOTE_DEV_HOST=${IP}" >> ~/.zshenv.tmp
+	echo 'export AUDIUS_REMOTE_DEV_HOST=$(curl -sfL -H "Metadata-Flavor: Google" http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip)' >> ~/.zshenv.tmp
```

Comment:
If we were to change it to a curl, we wouldn't need line 199.

However, the new curl only works for google while 199 uses the cli tools to get the IP address.